### DISCUSSION TITLE
I'm having a very strange problem that the Chakra-UI toast stops working after a browser page refresh

### DISCUSSION THREAD

**Discussion by GuilhermeAmado** [chakra-ui/chakra-ui#5470] [Q&A]
The fist time I open the page, the toast works normally, but after a  refresh (F5) the toast doesn't fire when its supposed to.
This is an application that relies on single-spa to manage various microfrontends. Chakra-UI is being used in 2 of these microfrontends. Maybe this has something to do with it?

I can't open an issue because I can't replicate this kind of environment on codesandbox.

**Comment by gsingh-ds** [comment]
@GuilhermeAmado  Did you find a fix for it?
